engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetFacesToCull.java

```diff
@@ -30,12 +29,12 @@
  * TODO: Add javadocs
  */
 public final class SetFacesToCull implements StateChange {
-
-    private static SetFacesToCull defaultInstance = new SetFacesToCull(GL_BACK); // also specified in OpenGL documentation
     private static Map<Integer, String> modeNameMap = ImmutableMap.of(GL_BACK, "GL_BACK",
             GL_FRONT, "GL_FRONT",
             GL_FRONT_AND_BACK, "GL_FRONT_AND_BACK");
-    private SetFacesToCullTask task;
+
+    private static SetFacesToCull defaultInstance = new SetFacesToCull(GL_BACK); // also specified in OpenGL documentation
```

Comment:
..I just moved code around, no extra information added from my side :P
But I'll take care of this comment! :D